crda/remotematcher.go

```diff
@@ -333,15 +332,22 @@ func (m *Matcher) invokeComponentAnalysesInBatch(ctx context.Context, records []
 		g.Go(func() error {
 			vulns, err := m.invokeComponentAnalyses(ctx, records[start:end])
 			if err != nil {
-				// log error
+				zlog.Error(ctx).Err(err).Msg("remote api call failure")
 				return nil
 			}
 			ctrlC <- vulns
 			return nil
 		})
 	}
 	g.Wait()
-	return ctrlC
+	close(ctrlC)
+
+	// Not sure how big
```

Comment:
This comment not longer makes sense here, can you remove it?

Reply:
yes.